pom.xml

```diff
@@ -77,6 +77,12 @@
             <version>1.10.9</version>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
```

Comment:
@arnaudroques not sure what you think about adding this as a test dependency?  It's not providing much for this PR.

Reply:
Well, we try to limit dependencies. So I think it's better if you can avoid it here.
Sorry about that, but thanks for your contribution !

Reply:
removed it